exchanges/gateio/gateio_test.go

```diff
@@ -2053,11 +2056,13 @@ func TestFuturesDataHandler(t *testing.T) {
 	g := new(Gateio) //nolint:govet // Intentional shadow to avoid future copy/paste mistakes
 	require.NoError(t, testexch.Setup(g), "Test instance Setup must not error")
 	testexch.FixtureToDataHandler(t, "testdata/wsFutures.json", func(m []byte) error { return g.WsHandleFuturesData(context.Background(), m, asset.CoinMarginedFutures) })
+	g.Websocket.DataHandler <- errors.New("hamster")
```

Comment:
Causing errors in CI [here](https://github.com/thrasher-corp/gocryptotrader/actions/runs/13306572368/job/37158821371#step:10:3146) probably just RM

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/4442c6e7e